packages/date-picker/src/shared/context/SharedDatePickerContext.utils.ts

```diff
@@ -13,16 +13,40 @@ import { BaseFontSize, Size } from '@leafygreen-ui/tokens';
 
 import { MAX_DATE, MIN_DATE } from '../constants';
 import { AutoComplete, BaseDatePickerProps, DatePickerState } from '../types';
+import { ModifiedPopoverProps } from '../types/BaseDatePickerProps.types';
 import { getFormatParts } from '../utils';
 
 import {
   SharedDatePickerContextProps,
   SharedDatePickerProviderProps,
 } from './SharedDatePickerContext.types';
 
+export type ModifiedPopoverPropkeys = keyof ModifiedPopoverProps;
+
 export type ContextPropKeys = keyof SharedDatePickerProviderProps &
   keyof BaseDatePickerProps;
 
+/**
+ * Prop names that are extended from popoverProps
+ * */
+export const modifiedPopoverPropNames: Array<ModifiedPopoverPropkeys> = [
+  'scrollContainer',
+  'portalContainer',
+  'portalClassName',
+  'align',
+  'justify',
+  'spacing',
+  'adjustOnMutation',
+  'popoverZIndex',
+  'onEnter',
+  'onEntering',
+  'onEntered',
+  'onExit',
+  'onExiting',
+  'onExited',
+  'contentClassName',
+];
```

Comment:
that might a good idea, so we can reuse this

Reply:
Maybe we can chat in person. I'm thinking about how this would make sense in `Popover`. This array of popover prop names is specific to DatePicker only, and its primary purpose is to filter out props using the `pickAndOmit` util. The list is too specific to live in `Popover`.  

My current thought process is whether we have an array of popover props available and whether we can filter them to what we need inside DatePicker.

Reply:
We _may_ need them elsewhere, but you're right, this is pretty specific to what we need for DatePicker